src/contracts/atlas/GasAccounting.sol

```diff
@@ -417,6 +420,20 @@ abstract contract GasAccounting is SafetyLocks {
         return (claimsPaidToBundler, netAtlasGasSurcharge);
     }
 
+    function _updateAnalytics(EscrowAccountAccessData memory aData, bool auctionWon, uint256 gasUsed) internal pure {
```

Comment:
maybe add a comment that this function won't be executed when `bidFind = true` so no risk of doublecounting

Reply:
Good call, added here https://github.com/FastLane-Labs/atlas/pull/330/commits/874354fa80e6a87fadad90beb086b47a7d25d632